packages/core/js-client/src/api.ts

```diff
@@ -204,13 +203,18 @@ export const v5_registerService = (
 
   // Wrapping service functions, selecting only those listed in schema, to convert their args js -> aqua and backwards
   const wrappedServiceImpl = Object.fromEntries(
-    Object.keys(serviceSchema).map((schemaKey) => {
+    Object.entries(serviceSchema).map(([schemaKey, schemaValue]) => {
+      const serviceImplValue = serviceImpl[schemaKey];
+
+      if (serviceImplValue === undefined) {
+        throw new Error(
+          `Expected to have ${schemaKey} in service implementation according to schema`,
```

Comment:
Does this happen if user didn't implement the service he was supposed to implement? Or when does it happen?

I would prefer a more actionable error. Users wouldn't know what's the schema what's the key and where, especially if they don't use typescript